src/lib/api/Testing.ts

```diff
@@ -0,0 +1,67 @@
+import axios, { AxiosInstance } from "axios";
+
+import ProjectsApi from "./resources/Projects";
+import InboxesApi from "./resources/Inboxes";
+import MessagesApi from "./resources/Messages";
+import encodeMailBuffers from "../mail-buffer-encoder";
+import handleSendingError from "../axios-logger";
+
+import CONFIG from "../../config";
+
+import { Mail, SendResponse } from "../../types/mailtrap";
+
+const { CLIENT_SETTINGS, ERRORS } = CONFIG;
+const { TESTING_ENDPOINT } = CLIENT_SETTINGS;
+
+const { TEST_INBOX_ID_MISSING, ACCOUNT_ID_MISSING } = ERRORS;
+
+export default class TestingAPI {
+  private client: AxiosInstance;
+
+  private testInboxId?: number;
+
+  private accountId?: number;
+
+  public projects: ProjectsApi;
+
+  public inboxes: InboxesApi;
+
+  public messages: MessagesApi;
+
+  constructor(client: AxiosInstance, testInboxId?: number, accountId?: number) {
+    this.client = client;
+    if (!testInboxId) {
+      // eslint-disable-next-line no-console
+      console.warn(TEST_INBOX_ID_MISSING);
+    }
+    this.testInboxId = testInboxId;
+    if (!accountId) {
```

Comment:
As I'm looking at this right now, we will always show this warning for users even if they don't want to use Testing Client
- #idea Maybe let's add in `src/lib/mailtrap-client.ts` lazy initialisation for testing client?
- wdyt?

Reply:
lazy loading will bring the following syntax:

```ts
const client = new MailtrapClient({ token: TOKEN, testInboxId: TEST_INBOX_ID, accountId: ACCOUNT_ID });
const testingClient = client.testing()

testingClient.projects.get()
```

Reply:
@VladimirTaytor I think we can introduce a getter method for testing and while users will try to access testing, they will receive warning messages. This way we don't have to call testing. Wdyt?